packages/webview_flutter/webview_flutter_android/lib/src/android_webview_controller.dart

```diff
@@ -731,6 +731,28 @@ class AndroidWebViewController extends PlatformWebViewController {
     _onJavaScriptPrompt = onJavaScriptTextInputDialog;
     return _webChromeClient.setSynchronousReturnValueForOnJsPrompt(true);
   }
+
+  @override
+  Future<void> setOverScrollMode(WebViewOverScrollMode mode) async {
+    switch (mode) {
+      case WebViewOverScrollMode.always:
+        return _webView.setOverScrollMode(
```

Comment:
Optional: if you use the newer `switch` style you can `return switch (mode) { ... }` and not have to replicate all the `return` calls.